rosdep/python.yaml

```
@@ -8495,6 +8495,22 @@ python3-qtpy:
   rhel:
     '8': [python3-QtPy]
   ubuntu: [python3-qtpy]
+python3-quick2wire-api-pip:
+  arch:
+    pip:
+      packages: [quick2wire-api]
+  debian:
+    pip:
+      packages: [quick2wire-api]
+  fedora:
+    pip:
+      packages: [quick2wire-api]
+  osx:
+    pip:
+      packages: [quick2wire-api]
+  ubuntu:
+    pip:
+      packages: [quick2wire-api]
```

Hi, please use the os-level wildcard for these rules:
```suggestion
python3-quick2wire-api-pip:
  '*':
    pip:
      packages: [quick2wire-api]
```